spec/support/e2e_setup.rb

```diff
@@ -48,6 +55,20 @@ def mui_select(choice, from:)
     find('li[role=option]', text: choice).click
   end
 
+  def mui_table_select(choice, row:, column:)
+    row_label = find('td', text: row)
+    scroll_to(row_label, align: :center)
+    column_label = find('th', text: column)
+    find("[aria-labelledby='#{row_label['id']} #{column_label['id']}']").click
+    find('li[role=option]', text: choice).click
+  end
+
+  def mui_date_select(label, date:)
+    field = find("[aria-label='#{label}']")
+    field.click
+    field.native.send_keys(:left, :left, :backspace, date.strftime('%m/%d/%Y'))
```

Comment:
This key sequence seems like a bit of magic. Maybe add a comment?

Reply:
Yes you're right -- sorry, I thought I'd left a PR comment to ask you about this, but I'd left it as a draft review!

I was stuck on this for some time, trying different things, and wasn't able to find another good way to interact with the datepicker. I thought it might be related to this issue [https://github.com/mui/mui-x/issues/4644] but it's not quite the same problem.